ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -73,6 +143,79 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
     static get is() {
       return 'nuxeo-checkbox-aggregation';
     }
+
+    static get properties() {
+      return {
+        _buckets: {
+          type: Array,
+          value: () => [],
+          computed: '_setBuckets(buckets)',
+        },
+        /**
+         * Make the element collapsible
+         */
+        collapsible: {
+          type: Boolean,
+          value: false,
+        },
+
+        /**
+         * Label of the element
+         */
+        label: {
+          type: String,
+          value: '',
+        },
+
+        _showAll: {
+          type: Boolean,
+          value: false,
+        },
+
+        _visibleBuckets: {
+          type: Array,
+          value: () => [],
+        },
+
+        /**
+         * If collapsible, the default visible items.
+         */
+        visibleItems: {
+          type: Number,
+          value: 8,
+        },
+      };
+    }
+
+    _isCollapsible(buckets) {
+      if (this.collapsible && buckets.length) {
+        return true;
+      }
+    }
+
+    _setBuckets(buckets) {
+      if (buckets.length) {
+        this._visibleBuckets = buckets.slice(0, this.visibleItems);
+        return buckets;
+      }
+    }
+
+    _toggleShow() {
+      this._showAll = !this._showAll;
+      if (this._showAll) {
+        this._visibleBuckets = this._buckets;
+      } else {
+        this._visibleBuckets = this._buckets.slice(0, this.visibleItems);
+      }
+    }
+
+    _computeShowMoreLabel() {
+      return this.i18n((this._showAll && 'checkboxAggregation.showLess') || 'checkboxAggregation.showMore');
+    }
+
+    _hideShowMore() {
+      return this._buckets && this._buckets.length <= this.visibleItems;
+    }
   }
 
   customElements.define(CheckboxAggregation.is, CheckboxAggregation);
```

Comment:
We don't need the line below anymore (`Nuxeo.CheckboxAggregation = CheckboxAggregation;`). 
If you want to perform the cleanup 😉